pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -205,18 +205,19 @@ func (c *Controller) nodeScopedListOptions() metav1.ListOptions {
 
 func (c *Controller) watchAppliedToGroups() {
 	// TODO: Cleanup AppliedToGroups that are removed during reconnection.
-	klog.Info("Start watching AppliedToGroups")
+	klog.Info("Starting watch for AppliedToGroups")
 	options := c.nodeScopedListOptions()
 	w, err := c.antreaClient.NetworkingV1beta1().AppliedToGroups().Watch(options)
 	if err != nil {
-		klog.Errorf("Failed to watch AppliedToGroups: %v", err)
+		klog.Errorf("error starting watch for AppliedToGroups: %v", err)
```

Comment:
should still be capitalized for a log message

Reply:
applies to other places below as well

Reply:
done, my mistake.